regression/testing_tool.py

```diff
@@ -381,3 +287,29 @@ def main():
 
 if __name__ == "__main__":
     main()
+
+# Utilities for REPL. This helps us to do batch changes over all tests.
+# Example:
+# 1. Invoke python3 in shell
+# 2. `import testing_tool`
+# 3. `testing_tool.apply_transform_over_tests(testing_tool.print_test)`
+
+# TODO We probably should obtain this from CMake somehow
+TEST_SUITES = [ "bitwuzla", "cvc", "mathsat", "smtlib", "z3", "esbmc-solidity", "Interval-analysis-ibex-contractor",
```

Comment:
Re comment: yes, we should. People create new sub.directories all the time.

Reply:
I guess we could change the transform function to work with a directory and use CMake to invoke a wrapper of the transform.

Reply:
Sounds complicated. As this is for manual use only, I'm fine leaving this list hardcoded for now. Maybe add a comment to check whether the list is still up to date for manual invocations of the new function.